app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">MCStatus</string>
+    <string name="about_version">Version %1$s (Build %2$d)</string>
+    <string name="action_refresh">Neuladen</string>
+    <string name="action_addserver">Server hinzufügen</string>
+    <string name="action_deleteserver">Server(s) löschen</string>
+    <string name="action_editserver">Server bearbeiten</string>
+    <string name="action_about">Über diese App</string>
+    <string name="action_help">Hilfe</string>
+    <string name="action_privacy">Privacy</string>
```

Comment:
@chaosfreak93 A German student I know suggests this one might want to be "Datenschutzerklärung", but she's a student and not a native speaker, so I'll defer to the native speaker on that.

Reply:
I think 'Datenschutz' sounds better